run_examples/run_ogcore_example.py

```diff
@@ -75,7 +75,7 @@ def main():
     ------------------------------------------------------------------------
     '''
     # update the effective corporate income tax rate
-    og_spec.update({'cit_rate': [0.35]})
+    og_spec.update({'cit_rate': [0.35], 'baseline_spending': False})
```

Comment:
@jdebacker. Why put `'zeta_D': [0.4], 'zeta_K': [0.1]` in line 52 and `'baseline_spending': False` in line 78 when these are the default values from `default_parameters.py`?

Reply:
@rickecon This can be removed.  I was playing with some values when running this script with K_g>0.

Reply:
@jdebacker. OK. I just took these extraneous values out and committed the change to my open PR to your branch.